sale_order_product_recommendation/wizards/sale_order_recommendation.py

```diff
@@ -220,11 +220,20 @@ def action_accept(self):
 class SaleOrderRecommendationLine(models.TransientModel):
     _name = "sale.order.recommendation.line"
     _description = "Recommended product for current sale order"
-    _order = "id"
+    _order = (
+        "product_priority desc, product_categ_id, units_delivered desc, "
+        "times_delivered desc, product_id, id"
+    )
```

Comment:
Before this patch, it was just ordered by id. That default order didn't seem very meaningful, that's why I changed it so freely, but I didn't mean to break the module.

I still think it makes sense to put first the favorites, isn't it?

So, the correct default order would be this?

```suggestion
    _order = "product_categ_id, units_delivered desc, times_delivered desc, id"
```

Or would it be just this other one?

```suggestion
    _order = "product_priority desc, id"
```

Reply:
I decided to go with the 2nd option, which is more behavior-preserving.

That way, this improvement can be merged faster. Still, on desktop view, users can click on the category header to be able to sort by category.

We will see which other options we have for implementing that order by default later.